### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nerfur**
removed dead link to telegram chat
add 3 more (global english, russian, chinese) telegram chats

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -174,7 +174,9 @@
 * `irc` [#OpenBSD](https://web.libera.chat/?channels=#openbsd) on irc.libera.chat
 * `irc` [#OpenBSD-Russian](http://webchat.freenode.net?channels=%23openbsd-russian) on irc.freenode.net
 * `matrix` [OpenBSD](https://matrix.to/#/%23openbsd%3Amatrix.org)
-* `telegram` [OpenBSD Jumpstart](https://t.me/joinchat/EzTjLQuG8Mcj89LYcDGKiQ)
```
Comment (ligurio) [inline README.md:177]: Valid link is https://t.me/joinchat/EzTjLQuG8MdUSVqFS1xA4w

**Review by ligurio — CHANGES_REQUESTED** [review]
Please update a patch

File: README.md [hunk 2]
```diff
@@ -174,7 +174,10 @@
...
+* `telegram` [OpenBSD](https://t.me/openbsd_en)
+* `telegram` [OpenBSD Jumpstart](https://t.me/joinchat/EzTjLQuG8MdUSVqFS1xA4w)
+* `telegram` [OpenBSD (russian)](https://t.me/openbsd_ru)
+* `telegram` [OpenBSD 中文结社 🐡](https://t.me/openbsd_zh)
```
Comment (ligurio) [inline README.md]: I would remove emodji and replace "中文结社" with "chinese" or "China" to make it consistent with other items.

File: README.md [hunk 3]
```diff
@@ -174,7 +174,10 @@
 * `irc` [#OpenBSD](https://web.libera.chat/?channels=#openbsd) on irc.libera.chat
 * `irc` [#OpenBSD-Russian](http://webchat.freenode.net?channels=%23openbsd-russian) on irc.freenode.net
 * `matrix` [OpenBSD](https://matrix.to/#/%23openbsd%3Amatrix.org)
-* `telegram` [OpenBSD Jumpstart](https://t.me/joinchat/EzTjLQuG8Mcj89LYcDGKiQ)
+* `telegram` [OpenBSD](https://t.me/openbsd_en)
+* `telegram` [OpenBSD Jumpstart](https://t.me/joinchat/EzTjLQuG8MdUSVqFS1xA4w)
+* `telegram` [OpenBSD (russian)](https://t.me/openbsd_ru)
```
Comment (ligurio) [inline README.md]: Round brackets can be removed

**Review by ligurio — CHANGES_REQUESTED** [review]

**Review by nerfur — COMMENTED** [review]
ok, btw, why not join our russian-language tg ? :D

File: README.md [hunk 4]
```diff
@@ -174,7 +174,10 @@
 * `irc` [#OpenBSD](https://web.libera.chat/?channels=#openbsd) on irc.libera.chat
 * `irc` [#OpenBSD-Russian](http://webchat.freenode.net?channels=%23openbsd-russian) on irc.freenode.net
 * `matrix` [OpenBSD](https://matrix.to/#/%23openbsd%3Amatrix.org)
-* `telegram` [OpenBSD Jumpstart](https://t.me/joinchat/EzTjLQuG8Mcj89LYcDGKiQ)
+* `telegram` [OpenBSD](https://t.me/openbsd_en)
```
Comment (ligurio) [inline README.md]: Shouldn't we specify here language too?

**Review by ligurio — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (ligurio) [reply README.md]: Fixed.

```diff
--- a/README.md
+++ b/README.md
@@ -174,10 +174,10 @@
 * `irc` [#OpenBSD](https://web.libera.chat/?channels=#openbsd) on irc.libera.chat
 * `irc` [#OpenBSD-Russian](http://webchat.freenode.net?channels=%23openbsd-russian) on irc.freenode.net
 * `matrix` [OpenBSD](https://matrix.to/#/%23openbsd%3Amatrix.org)
-* `telegram` [OpenBSD](https://t.me/openbsd_en)
 * `telegram` [OpenBSD Jumpstart](https://t.me/joinchat/EzTjLQuG8MdUSVqFS1xA4w)
-* `telegram` [OpenBSD russian](https://t.me/openbsd_ru)
-* `telegram` [OpenBSD chinese](https://t.me/openbsd_zh)
+* `telegram` [OpenBSD English](https://t.me/openbsd_en)
+* `telegram` [OpenBSD Russian](https://t.me/openbsd_ru)
+* `telegram` [OpenBSD Chinese](https://t.me/openbsd_zh)
 * `telegram` [OpenBSD Brazil](https://t.me/OpenBSDbr)
 * `telegram` [OpenBSD Spain](https://t.me/OpenBSD_es)
 * `telegram` [BSDar Argentina](https://t.me/BSDar)
```

**Pull request merged**